python/cudf/cudf/core/dataframe.py

```diff
@@ -3544,6 +3544,22 @@ def std(self, **kwargs):
     def var(self, **kwargs):
         return self._apply_support_method("var", **kwargs)
 
+    def kurtosis(self, numeric_only=None, **kwargs):
```

Comment:
Can you add the additional kwargs for API compatibility and then ensure that if they're non-default we throw an exception?